packages/components/src/components/Box/Box.tsx

```diff
@@ -14,7 +14,10 @@ export const Box = React.forwardRef((props, ref) => {
 
   const {atomProps, nativeProps} = splitProps(restProps);
 
-  const atomicClasses = atoms(atomProps);
+  const atomicClasses = atoms({
+    reset: typeof Component === 'string' ? Component : 'div',
```

Comment:
```suggestion
    ...(typeof Component === 'string' && {reset: Component}),
```

**thought:** Perhaps we should only apply the reset if the value is a string. This way we're not apply `div` resets for a component that could return any element.